cli/cmd/inject.go

```diff
@@ -37,6 +37,7 @@ const (
 )
 
 type injectOptions struct {
+	noInitContainer bool
 	*proxyConfigOptions
```

Comment:
I had considered putting it there but decided against it because it isn’t really a config option of the proxy itself like the other options. Thoughts?